src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Kubernetes Summary page
+tags:
+  - APM
+  - APM UI pages
+  - Kubernetes
+metaDescription: The Kubernetes Summary page provides charts and tables that you can drill down into details about your selected app's performance.
+---
+
+The Kubernetes APM **Summary** page provides general information about the selected cluster, including the age of the current deployment, the number of pods available, CPU and memory usage, pod logs, and metric comparison.
+
+![This is an image of the Kubernetes APM page](./images/kubernetes-apm-ui.png "Kubernetes page")
+
+<figcaption>[one.newrelic.com](one.newrelic.com) > **Explorer** > Select your cluster > **Kubernetes**</figcaption>
+
+<Callout variant="tip">
+  To get a high-level overview of all your applications and services, use the [New Relic explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer).
+</Callout>
+
+## Monitor your Kubernetes cluster [#monitor-kubernetes]
+
+The Kubernetes page in APM offers the following information about your Kubernetes cluster:
+
+<CollapserGroup>
+    <Collapser
+    id="header"
+    title="Header"
+    >
+    
+    You can see your cluster name, deployment, and namespace. If you click the cluster name and the deployment, you'll get some details about each of them in a pop-up window.
+
+    In the **Instances** drop-down, you can select a single pod instance of your service.   
+
+    ![This is an image of the Kubernetes APM page](./images/kubernetes-header.png "Kubernetes page - Header")
+
+    </Collapser>
+    <Collapser
+    id="current-deployment"
+    title="Current deployment"
+    >
+    You can check the age of the kubernetes deployment, how many pods are available or desired, and the pods that are missing.
```

Comment:
Might want to add a note that the age of Deployment is when the deployment was first created.